pkg/config/crd/store.go

```diff
@@ -138,17 +151,20 @@ func (s *Store) checkAndCreateCaches(
 		}
 		s.cacheMutex.Lock()
 		for _, res := range resources.APIResources {
-			if _, ok := s.caches[res.Kind]; ok {
+			if _, ok := s.caches[cacheKey{res.Kind, checkNs}]; ok {
```

Comment:
`checkNs` is not a loop variable, why is the 1st element of the namespaces special? why is it checked here in a loop?

Reply:
removed, reformed the loop structure a bit.